scripts/scripts.js

```diff
@@ -192,7 +193,16 @@ export function addPhotoCredit(pictures) {
  * @return {string|null}
  */
 function findTemplate(className) {
-  return Object.values(ARTICLE_TEMPLATES).find((template) => template === className);
+  return Object.values(ARTICLE_TEMPLATES).find((template) => {
+    if (!className.startsWith(template)) {
+      return false;
+    }
+    // Adding base template class to body because any queries added to the
+    // template metadata entry are combine which will cause CSS issues.
+    // example: "Template (sheet query)" has a class of: "template-sheet-query"
+    document.body.classList.add(template);
```

Comment:
I think this should not be done in `findTemplate` since the function as its name implies should only find the template, not have side effects like adding a class. Maybe you can do that rather after the function call if a template is found ?

Reply:
this was moved out of the findTemplate function.
The findTemplate function no longer exists as it is not used with the new version of buildTemplate